cluster-autoscaler/cloudprovider/gce/autoscaling_gce_client_test.go

```diff
@@ -143,31 +143,43 @@ func TestErrors(t *testing.T) {
 
 	testCases := []struct {
 		errorCodes         []string
+		errorMessage       string
 		expectedErrorCode  string
 		expectedErrorClass cloudprovider.InstanceErrorClass
 	}{
 		{
 			errorCodes:         []string{"IP_SPACE_EXHAUSTED"},
+			errorMessage:       "",
```

Comment:
nit: empty string is the zero value for string, so it will be the same if you don't set it. I think the convention is to skip irrelevant parameters in test cases (it's pretty normal for some parameters to only be relevant for some cases).

Reply:
Updated.